### DISCUSSION TITLE
February 15th, 2023

### DISCUSSION THREAD

**Discussion by LukasBuecherl** [SynBioDex/SBOL-Editors#45] [Editors meetings]
# Participants
* Chris Myers
* James Scott-Brown
* Tom Mitchell
* Gonzalo Vidal
* Carolus Vitalis
* Lukas Buecherl

# Agenda

## Open Discussion
* Harmony, SEED, and IWBDA goals  
* Harmony April 24th to 27th (Discuss SEP 57)

## Website (James S-B) 
* 

## SEPs / Github issues (Gonzalo)
* [SEP 57](https://github.com/SynBioDex/SEPs/pull/123): Where would the new terms be located
* Terms must be approved Best Practices 
* Are Best Practices tied to a specific SBOL Version
* Focus of next steering committee meeting 

## Communications (Carolus)
* Focus on Twitter posts
* Created Content for next few weeks

## Report on other SBOL meetings (Tom)
*

**Comment by Gonza10V** [comment]
For HARMONY I think we can propose  2 breakout rooms:
 - SEP057 
 - SEED workshop preparation
